packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -2203,6 +2203,18 @@ export class LitNodeClientNodeJs
 
     const signatures: SessionSigsMap = {};
 
+    if (!this.ready) {
```

Comment:
Okay `getSessionSig()` was the only function missing `this.ready` check. Actually we check this in the `signSessionKey()`?

Reply:
We do check it in `signSessionKey()`, but that method isn't called from `getSessionSigs()` -- it is only called from the `authNeededCallback()` defined in `getPkpSessionSigs()` and in some auth providers -- so I added it here for completeness :+1: